spark-dependencies/pom.xml

```diff
@@ -776,6 +818,14 @@
     </profile>
 
     <profile>
+      <!--only used for pyspark/sparkR needed spark dist is downloaded, no need to download auto-->
+      <id>spark-dist-cached</id>
```

Comment:
validate phrase is OK. only make sure let download-maven-plugin don't trigger download as it's so slow developer can download spark dist manual.